Ports: various dresses/some jean variations and added to loadout. & chaps and corsets added to kinkmate

Commandersand opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
more clothing! also fixed athletic shorts for loadout
also i need clarification on what icons i needed to set, item_color/states. 
idk which ones i needed and which ones i didn't so something's broken
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
customization options also fixes a bug i made haha
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Ported clothing from RP station, check uniform loadout and kinkmate
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

izzyinbox commented on code/modules/clothing/under/shorts.dm:
```diff
@@ -35,4 +35,22 @@
+	name = "short denim skirt"
+	desc = "Who likes short skirts? we like short skirts!"
+	icon_state = "denim_skirt
+	item_color = "denim_skirt
```
Missing some close quotation marks around here.

izzyinbox commented on modular_citadel/code/modules/client/loadout/uniform.dm:
```diff
@@ -78,6 +78,11 @@
 	path = /obj/item/clothing/under/plaid_skirt
 
+/datum/gear/sweptskirt
+	name = "Swept skirt
```
And here too.
```suggestion
	name = "Swept skirt"
```

r4d6 committed:
Add a nose boop sound

r4d6 committed:
Add a boop sound effect

r4d6 committed:
Make the boop louder.

Ghommie committed:
Fixes pubby wall meds being var-edited snowflakes.

r4d6 committed:
Commit suggestions

Co-Authored-By: deathride58 <deathride58@users.noreply.github.com>

r4d6 committed:
Commit suggestions

Co-Authored-By: deathride58 <deathride58@users.noreply.github.com>

r4d6 committed:
Commit suggestions

Co-Authored-By: deathride58 <deathride58@users.noreply.github.com>

r4d6 committed:
Commit suggestions

Co-Authored-By: deathride58 <deathride58@users.noreply.github.com>

r4d6 committed:
Commit suggestions

Co-Authored-By: deathride58 <deathride58@users.noreply.github.com>

r4d6 committed:
Commit suggestions

Co-Authored-By: deathride58 <deathride58@users.noreply.github.com>

Ghommie committed:
Fixes AI being unable to talk through the holopad.

Ghommie committed:
Travis had a stroke.

Ghommie committed:
ports "Standardize obj/updateicon() and update_icons() to update_icon()"

silicons committed:
Update supply.dm

Putnam3145 committed:
Merge pull request #10208 from Ghommie/Ghommie-cit469

Fixes AI being unable to talk through the holopad.

Trilbyspaceclone committed:
Update code/modules/vending/cola.dm

Co-Authored-By: Ghom <42542238+Ghommie@users.noreply.github.com>

Ghommie committed:
Brings carbon action intent hotkeys down to living.

Trilbyspaceclone committed:
Update drink_reagents.dm

Linzolle committed:
Merge pull request #10180 from Ghommie/Ghommie-cit462

[s] virtual reality component improvements, fixes and documentation.

CitadelStationBot committed:
Automatic changelog generation for PR #10180 [ci skip]

Linzolle committed:
Merge pull request #10181 from Ghommie/Ghommie-cit463

Fixing a few pill bottle issues with the ChemMaster.

CitadelStationBot committed:
Automatic changelog generation for PR #10181 [ci skip]

Ghommie committed:
Porting a few martial art code updates.

Ghommie committed:
Merge branch 'master' of https://github.com/Citadel-Station-13/Citadel-Station-13 into Ghommie-cit453

GrayRachnid committed:
Adds gas miners to all maps.

BlackMajor committed:
Compress secK9

BlackMajor committed:
whoops

Ghommie committed:
Hopefully this won't screw up travis.

Ghommie committed:
Two slimepeople fixes.

BlackMajor committed:
Merge pull request #10072 from Useroth/new-tg-plants

Ported bamboo stuff from tg.

CitadelStationBot committed:
Automatic changelog generation for PR #10072 [ci skip]

Putnam3145 committed:
Merge pull request #10225 from Ghommie/Ghommie-cit475

Two slimepeople fixes.

CitadelStationBot committed:
Automatic changelog generation for PR #10225 [ci skip]

Trilbyspaceclone committed:
Update drink_reagents.dm

Putnam3145 committed:
Merge pull request #10209 from Ghommie/Ghommie-cit470

ports "Standardize obj/updateicon() and update_icons() to update_icon()"

Putnam3145 committed:
Merge pull request #10203 from DeltaFire15/mechaccess

Nukies and deathsquaddies can now finally fix their mechs internal fires

CitadelStationBot committed:
Automatic changelog generation for PR #10203 [ci skip]

Putnam3145 committed:
Merge pull request #10157 from Ghommie/Ghommie-cit452

Ports "Fixes powernet avail checks"

CitadelStationBot committed:
Automatic changelog generation for PR #10157 [ci skip]

Putnam3145 committed:
Merge pull request #10155 from Seris02/windowfix

adds some distance checks to some things

CitadelStationBot committed:
Automatic changelog generation for PR #10155 [ci skip]

Putnam3145 committed:
Merge pull request #10111 from kappa-sama/patch-11

send help please, or how i vowed to kill kevinz (you can now strip people while aggrograbbing)

CitadelStationBot committed:
Automatic changelog generation for PR #10111 [ci skip]

Putnam3145 committed:
Merge pull request #10066 from Sishen1542/aaaaaaaa

neovgre qol changes

Putnam3145 committed:
Merge pull request #10216 from Ghommie/Ghommie-cit474

Porting a few martial art code updates.

CitadelStationBot committed:
Automatic changelog generation for PR #10216 [ci skip]

Putnam3145 committed:
Merge pull request #10172 from Ghommie/Ghommie-cit460

Porting several quirks code updates.

CitadelStationBot committed:
Automatic changelog generation for PR #10172 [ci skip]

Putnam3145 committed:
Merge pull request #10167 from Ghommie/Ghommie-cit458

Fixing some not safe for work oddity.

Putnam3145 committed:
Merge pull request #10026 from Linzolle/neck-slice

Neck slice

CitadelStationBot committed:
Automatic changelog generation for PR #10026 [ci skip]

Putnam3145 committed:
Merge pull request #10076 from Ghommie/Ghommie-cit437

Fixes alien tech node not being unlockable with subtypes of the accepted items.

CitadelStationBot committed:
Automatic changelog generation for PR #10076 [ci skip]

Trilbyspaceclone committed:
Update drinks.dm

Putnam3145 committed:
Merge pull request #10153 from Linzolle/fireman-clarity

make unable to carry fireman message more clear when unable to carry non-humans

CitadelStationBot committed:
Automatic changelog generation for PR #10153 [ci skip]

Trilbyspaceclone committed:
Merge branch 'master' into donut

Trilbyspaceclone committed:
??

Trilbyspaceclone committed:
Update snacks.dm

Commit:
oops

Linzolle committed:
Merge branch 'master' of https://github.com/Citadel-Station-13/Citadel-Station-13 into abductor-update

Putnam3145 committed:
probably fixes only neutral traits existing

silicons committed:
Merge pull request #10227 from Putnam3145/quirk-fix

Probably fixes only neutral traits existing

CitadelStationBot committed:
Automatic changelog generation for PR #10227 [ci skip]

Putnam3145 committed:
argk

B1gb3ast committed:
Merge pull request #1 from Citadel-Station-13/master

update

B1gb3ast committed:
Chef buffs chef buffs tastier food.

"The strong flavor of" was always in game but someone put it behind an else if statement that would never proc.
Now if there's more than 45% (or 30% if you are a lizzard, hissss) of a reagent in a beaker, you will taste a strong flavor! 
Hurrah!

silicons committed:
Merge pull request #10228 from Putnam3145/quirk-fix

fixes quirks (again)

CitadelStationBot committed:
Automatic changelog generation for PR #10228 [ci skip]

Seris02 committed:
fixes the sprites

Ghommie committed:
Merge branch 'master' of https://github.com/Citadel-Station-13/Citadel-Station-13 into Ghommie-cit320

B1gb3ast committed:
Actually hold on

So that not everything in the game now tastes like STRONG FLAVOUR. That'd be annoying.

DeltaFire15 committed:
Magic number man bad, also all heads can spam CC now

B1gb3ast committed:
im baby dumb and removed "a hint of" by accident.

Okay whoops haha there we go now both strong and weak should work.

B1gb3ast committed:
oh hey macros.

slightly less ugly ™

B1gb3ast committed:
modular man bad.

now mörge this so chefs can make delicious dish, hemk.

lolman360 committed:
Update carbon.dm

lolman360 committed:
lolman acctually knows how the stuff work

lolman360 committed:
AAAAAAAAAAAAAAAAAAAA

silicons committed:
Update flashbang.dm

silicons committed:
Merge pull request #10223 from BlackMajor/Yeehaw

Removes Security K9 dogborg. The last of the infinity dogborg removals. (same treatment as the others)

silicons committed:
thank you based ghomtainer (when)

Co-Authored-By: Ghom <42542238+Ghommie@users.noreply.github.com>

silicons committed:
thank you based ghomtainer (when) 2x how did i not notice this

silicons committed:
Merge pull request #10204 from r4d6/patch-2

Port boops and handshakes

CitadelStationBot committed:
Automatic changelog generation for PR #10204 [ci skip]

silicons committed:
Merge pull request #10229 from B1gb3ast/master

Fixes nonfunctional code so that tongues can taste strong flavors

Ghommie committed:
Converts the main timsorts file from relative pathing to absolute pathing.

Ghommie committed:
Ports "Added TRAIT_DISK_VERIFIER, refactors how fake disk checks work slightly"

Ghommie committed:
wew

nik707 committed:
Merge pull request #10084 from Hatterhat/i-dont-test-my-code

i dont make good code

nik707 committed:
Merge pull request #10090 from Hatterhat/the-one-where-a-4-tc-nukies-kit-gets-buffed

the one where the combat medic kit gets buffed

CitadelStationBot committed:
Automatic changelog generation for PR #10090 [ci skip]

r4d6 committed:
Add Engines submaps

r4d6 committed:
Update Boxstation to have Engines landmark

r4d6 committed:
Adding a list for the stations rooms

r4d6 committed:
Fix a possible alert

r4d6 committed:
Add the stuff to seed the station with rooms

r4d6 committed:
Add the landmarks

r4d6 committed:
Add the file for making the submaps actual ruins

r4d6 committed:
Merge pull request #1 from r4d6/master

Wrong branch

r4d6 committed:
Add the sprite for the landmark

r4d6 committed:
Make it include my files

nik707 committed:
Merge pull request #10067 from Sishen1542/botnayn

You can change a plant's name, description and its seed's description by using a pen on the seed

CitadelStationBot committed:
Automatic changelog generation for PR #10067 [ci skip]

Linzolle committed:
Merge pull request #10160 from Ghommie/Ghommie-cit453

Ports improved and simplier fire alarms.

CitadelStationBot committed:
Automatic changelog generation for PR #10160 [ci skip]

Linzolle committed:
Merge pull request #9722 from Ghommie/Ghommie-cit320

Umodularize saycode and meat slabs.

Linzolle committed:
drop

Linzolle committed:
Merge pull request #9910 from Ghommie/Ghommie-cit390

Unmodularize gamemodes folder.

Linzolle committed:
Merge pull request #10079 from Ghommie/Ghommie-cit438

Fixes reactive armor onmob overlays not updating when toggled and reactive teleport armor still using forceMove()

CitadelStationBot committed:
Automatic changelog generation for PR #10079 [ci skip]

Linzolle committed:
Merge pull request #10199 from Ghommie/Ghommie-cit467

Fixing and cleaning up the blood bank generator code.

Linzolle committed:
Merge pull request #10207 from Ghommie/Ghommie-cit468

Fixes pubby wall meds being var-edited snowflakes.

CitadelStationBot committed:
Automatic changelog generation for PR #10207 [ci skip]

Linzolle committed:
Merge pull request #10214 from Ghommie/Ghommie-cit472

Brings carbon action intent hotkeys down to living.

CitadelStationBot committed:
Automatic changelog generation for PR #10214 [ci skip]

Linzolle committed:
Merge pull request #10173 from Ghommie/Ghommie-cit461

Fixing a boxstation pipenet mapping issue.

silicons committed:
Update mob.dm

silicons committed:
Update mob.dm

Commit:
Merge remote-tracking branch 'upstream/master' into substationV2

Commit:
Redo my changes on Box

Hopefully this will fix the conflict with the fire alarms.

lolman360 committed:
h

Ghommie committed:
Merge pull request #10193 from Commandersand/yeet

gamer is now filtered for clowns

CitadelStationBot committed:
Automatic changelog generation for PR #10193 [ci skip]

Ghommie committed:
I'm a god. How can you defeat a god? What a grand and intoxicating innocence, how can you be so naive?

Ghommie committed:
Merge pull request #10054 from kappa-sama/patch-10

autolathe .357 speedloaders->bullets, surplus rifle bullets in hacked lathe

CitadelStationBot committed:
Automatic changelog generation for PR #10054 [ci skip]

Ghommie committed:
Merge pull request #10126 from Putnam3145/he-tweaks

Makes HE pipes realistically radiate away heat

CitadelStationBot committed:
Automatic changelog generation for PR #10126 [ci skip]

Ghommie committed:
Merge pull request #10142 from Trilbyspaceclone/patch-244

Feast or Famine is shit. Makes CE hardsuit raditation proof hat wise, buffes its rest of body by 5

CitadelStationBot committed:
Automatic changelog generation for PR #10142 [ci skip]

Ghommie committed:
Merge pull request #10051 from kappa-sama/patch-9

puts our legion more up to date with tg's

CitadelStationBot committed:
Automatic changelog generation for PR #10051 [ci skip]

Ghommie committed:
Merge pull request #9925 from Linzolle/cosmic-brain

change cosmic coat crafting recipe

CitadelStationBot committed:
Automatic changelog generation for PR #9925 [ci skip]

Ghommie committed:
Merge pull request #9888 from Citadel-Station-13/kevinz000-patch-116

The high voltage from tasers can now set people with fire stacks (so things like dousing in fuel) on fire

Ghommie committed:
Merge pull request #9563 from Thalpy/Superbases-goodbye

[TESTMERGE] Goodbye, superpH, you were too wild for our world.

CitadelStationBot committed:
Automatic changelog generation for PR #9563 [ci skip]

Ghommie committed:
Merge pull request #9798 from Citadel-Station-13/kevinz000-patch-114

Radiation storms no longer do random mutations in of itself, but are far more lethal

DeltaFire15 committed:
Atomises out the magicnumber change

Ghommie committed:
Merge pull request #10231 from Seris02/wintercoatcolors

fixes the sprites for some winter coats

CitadelStationBot committed:
Automatic changelog generation for PR #10231 [ci skip]

Ghommie committed:
Merge pull request #10235 from Citadel-Station-13/kevinz000-patch-128

flashbangs process flash/bang separately based on blindness/hearing and uses viewers(), which means xray users suffer too if they see them go off

CitadelStationBot committed:
Automatic changelog generation for PR #10235 [ci skip]

Ghommie committed:
Merge pull request #10200 from Citadel-Station-13/kevinz000-patch-125

Adds 8 character save slots

CitadelStationBot committed:
Automatic changelog generation for PR #10200 [ci skip]

r4d6 committed:
Fix the nose boop.

r4d6 committed:
Remove extra indentations

lolman360 committed:
Update inhand_holder.dm

lolman360 committed:
hhh

lolman360 committed:
ghom help

lolman360 committed:
ff

lolman360 committed:
ff

silicons committed:
Merge pull request #10058 from Ghommie/Ghommie-cit432

Spinfusor nerf, cause a 7k cost supply pack containing 32 explosive munitions that work just fine without the gun is just pure balance.

CitadelStationBot committed:
Automatic changelog generation for PR #10058 [ci skip]

Ghommie committed:
Merge pull request #10059 from lolman360/HeadCorgi

you can pick up corgis now

CitadelStationBot committed:
Automatic changelog generation for PR #10059 [ci skip]

Ghommie committed:
Merge pull request #9596 from Linzolle/abductor-update

Bring abductors up-to-date with /tg/.

CitadelStationBot committed:
Automatic changelog generation for PR #9596 [ci skip]

Ghommie committed:
Merge pull request #9940 from Putnam3145/dynamic-roundstart-overhaul

Dynamic rulesets that have fired in recent rounds have lower weight

CitadelStationBot committed:
Automatic changelog generation for PR #9940 [ci skip]

Ghommie committed:
Merge pull request #10232 from DeltaFire15/cc-headmessages

All heads of staff can now spam CC.

CitadelStationBot committed:
Automatic changelog generation for PR #10232 [ci skip]

Ghommie committed:
Merge pull request #10141 from Putnam3145/dynamic-statbus

Adds a bunch of statbus logging to dynamic

CitadelStationBot committed:
Automatic changelog generation for PR #10141 [ci skip]

Ghommie committed:
Begone crap.

Ghommie committed:
Merge pull request #10077 from ForrestWick/uplinkchanges

Moves brainwash surgery from role restricted to devices and tools in the uplink

Ghommie committed:
I gotta go fast.

Ghommie committed:
Merge pull request #10012 from Commandersand/INEEDAMEDICBAG

Rebalance/port: refactor ported, ointment/brute pack healing nerf but max stack doubled. gauze heals a bit now

Commit:
QoL Update

Move the Engines to a subfolder to be cleaner when there will be 20 submaps on different maps and change the Tesla Submap to incorporate a few changes I did for the Singulo submap.

DeltaFire15 committed:
Magic number baaad

Commit:
Make things even more organized

Ghommie committed:
Better safe than sorry.

Ghommie committed:
Merge pull request #10164 from kappa-sama/martialartist

What ho, Muscle Wizard! Might you cast us a spell?

CitadelStationBot committed:
Automatic changelog generation for PR #10164 [ci skip]

Ghommie committed:
Merge pull request #10120 from Citadel-Station-13/night_vision

Night vision trait but it just boosts barely-lit visibility

CitadelStationBot committed:
Automatic changelog generation for PR #10120 [ci skip]

Ghommie committed:
Merge pull request #10257 from DeltaFire15/communication-magicnumbad

Removes a magicnumber

CitadelStationBot committed:
Automatic changelog generation for PR #10257 [ci skip]

Ghommie committed:
typo

Ghommie committed:
Unnecessary OOC note.

Ghommie committed:
Is merging someone else's PR you personally fixed considerable as self-merging?

Ghommie committed:
I'm playing dirty because we have too many PRs to merge anyway.

Ghommie committed:
Merge pull request #10096 from Citadel-Station-13/kevinz000-patch-123

nanites can no longer spread through atmospherics-blocking objects

CitadelStationBot committed:
Automatic changelog generation for PR #10096 [ci skip]

Ghommie committed:
Merge pull request #9950 from Citadel-Station-13/marauder-nerf-1

Nerfs the Clockwork Marauder meta

CitadelStationBot committed:
Automatic changelog generation for PR #9950 [ci skip]

Ghommie committed:
Merge pull request #10190 from Trilbyspaceclone/honeysoda

The new buzz around the new soft drink

CitadelStationBot committed:
Automatic changelog generation for PR #10190 [ci skip]

Ghommie committed:
Merge pull request #9830 from DeltaFire15/cbt-clockvsbloodtorture

Stun warfare rebalance - CBT edition

CitadelStationBot committed:
Automatic changelog generation for PR #9830 [ci skip]

Commit:
Suggestions

Commit:
Fix indentation

silicons committed:
lowered to 10 after i struggled to open tickets panel

Linzolle committed:
Merge pull request #10241 from Ghommie/Ghommie-cit482

 Ports "Added TRAIT_DISK_VERIFIER, refactors how fake disk checks work slightly

LetterJay committed:
FUUUUUUUUUUUCK

Commit:
bazinga

e

Trilbyspaceclone committed:
Nerfs fun

LetterJay committed:
FUUUUUUUUUCK v2

Putnam3145 committed:
Merge pull request #10263 from Trilbyspaceclone/patch-247

Makes Buzz Fuzz less coma'enducing

CitadelStationBot committed:
Automatic changelog generation for PR #10263 [ci skip]

Commit:
QoL Update

- Rename the cameras and place two more

- Unanchor the shields generators and emitters because they were bugged and will save more time

- Put more rad closets on the Singulo

- Put more insuls on the Tesla since they protect

- Moved the Engineer clothing locker to Boxstation instead of having it in the Engine

- Added a button to the shutters on the north side of PA

- Added a marker to show where to place the console

- Rewire the power wire to be more in a straight line.

Commit:
Forgot a "

Commit:
Forgot a button on the Tesla

Trilbyspaceclone committed:
Merge branch 'master' into donut

silicons committed:
Update supply.dm

Trilbyspaceclone committed:
readds sprites

Ghommie committed:
Fills white first aid kits to the brim. Ergo how I was fooled into believing it was a tweak and not a nerf.

Ghommie committed:
Fix goose meat.

Commit:
Connect the fuel pipe to somewhere

Commit:
no more 40% time dilation

LetterJay committed:
Update __donator.dm

LetterJay committed:
Update modular_citadel/code/modules/client/loadout/__donator.dm

Co-Authored-By: Ghom <42542238+Ghommie@users.noreply.github.com>

LetterJay committed:
Update modular_citadel/code/modules/client/loadout/__donator.dm

Co-Authored-By: Ghom <42542238+Ghommie@users.noreply.github.com>

LetterJay committed:
Update modular_citadel/code/modules/client/loadout/__donator.dm

Co-Authored-By: Ghom <42542238+Ghommie@users.noreply.github.com>

LetterJay committed:
Update custom_items.dm

silicons committed:
Merge pull request #10254 from r4d6/patch-1

Fix the nose boop

CitadelStationBot committed:
Automatic changelog generation for PR #10254 [ci skip]

Commit:
Fix areas and tiles getting ripped off

silicons committed:
Merge pull request #10240 from Ghommie/Ghommie-cit481

Converts the main timsorts file from relative pathing to absolute pathing.

Commit:
Also give Tesla's emitters R-floors

Ghommie committed:
Merge pull request #10093 from Trilbyspaceclone/donut

[Ready] Hey why dont I port that one donut pr? Ye just the one food related pr... Just the one... Oh no

CitadelStationBot committed:
Automatic changelog generation for PR #10093 [ci skip]

Ghommie committed:
Merge pull request #10210 from Citadel-Station-13/kevinz000-patch-127

Cargo shuttle now silently leaves behind slaughter demons + revenants

CitadelStationBot committed:
Automatic changelog generation for PR #10210 [ci skip]

Ghommie committed:
Merge pull request #10246 from Citadel-Station-13/kevinz000-patch-129

Makes mob/Stat on a 1 second timer rather than fast-as-possible to tone down highpop lag

CitadelStationBot committed:
Automatic changelog generation for PR #10246 [ci skip]

deathride58 committed:
Update custom_items.dm

Linzolle committed:
Merge pull request #10139 from Citadel-Station-13/donator

Donator Item - Ready For Review

silicons committed:
Merge pull request #10267 from Ghommie/Ghommie-cit486

Fills white first aid kits to the brim. Ergo how I was fooled into believing it was a tweak and not a nerf.

silicons committed:
Merge pull request #10218 from GrayRachnid/ADDS-MINERS

Adds gas miners to all maps.

CitadelStationBot committed:
Automatic changelog generation for PR #10218 [ci skip]

Ghommie committed:
Merge pull request #10274 from kappa-sama/kappa-sama-patch-1

no more 40% time dilation from broodmother

CitadelStationBot committed:
Automatic changelog generation for PR #10274 [ci skip]

Commit:
Add the option to restrict the Engine that spawn

Commit:
Revert "Add the option to restrict the Engine that spawn"

This reverts commit e3e1f9163538fb221a0697f6e36448032b3199d5.

Commit:
Revert "Revert "Add the option to restrict the Engine that spawn""

This reverts commit d851beea169841569078665b771b6f154700e26e.

Commit:
Add config options

Commit:
Move the config up to avoid conflict

Ghommie committed:
Merge pull request #10245 from r4d6/substationV2

Make Box's Engine be randomised between SM, Tesla and Singulo V2 [TEST-MERGE]

CitadelStationBot committed:
Automatic changelog generation for PR #10245 [ci skip]

Linzolle committed:
Merge pull request #10269 from Ghommie/Ghommie-cit488

Fix goose meat.

CitadelStationBot committed:
Automatic changelog generation for PR #10269 [ci skip]

Seris02 committed:
Merge branch 'master' of https://github.com/Citadel-Station-13/Citadel-Station-13 into rising_bass

Seris02 committed:
Merge branch 'rising_bass' of https://github.com/Seris02/Citadel-Station-13 into rising_bass

Seris02 committed:
a few fixes here and there and updating it

Ghommie committed:
Merge pull request #9751 from Seris02/rising_bass

RISING BASS, THE NEW MARTIAL ARTS

CitadelStationBot committed:
Automatic changelog generation for PR #9751 [ci skip]

cowsLUL committed:
Merge remote-tracking branch 'origin/hugebasketballshooorts' into localshorts

# Conflicts:
#	icons/mob/uniform.dmi
#	icons/mob/uniform_digi.dmi
#	icons/obj/clothing/uniforms.dmi

cowsLUL committed:
this either broke or fixed everything
